src/main/java/com/ibm/cics/bundle/parts/BundlePublisher.java

```diff
@@ -63,13 +63,13 @@ public class BundlePublisher {
 	private final int major;
 	private final int minor;
 	private final int micro;
-	private final int release;
+	private final long release;
 	
 	private Consumer<Path> listener = f -> {};
 	private Map<Path, BundleResource> bundleResources = new HashMap<>();
 	private List<Define> defines = new ArrayList<>();
 
-	public BundlePublisher(Path bundleRoot, String bundleId, int major, int minor, int micro, int release) {
```

Comment:
How about instead of changing these constructor signatures from `int` to `long` to accommodate Maven, why don't we add some new constructors which accept `long` and have the `int` constructors widen to `long` and delegate to them, that way we'll preserve binary compatibility and not have to bump the version number.  We could also consider deprecating the `int` version of the constructor, but I don't really see the harm.

I might be second-guessing myself here, but we use that value to set the `bundleRelease` manifest header right?  This error message doc seems to indicate that the value must be `0` and that makes me think that it's supposed to reference the version of the CICS bundle schema we're adhering to, rather than for our bundle's version information.

Reply:
I have removed the bundleRelease parameter from the constructor, and just set it in the writeManifest function as it will always be 0. This requires a major version bump though, and a test needed updating which I have done. 

The gradle and maven plugin have been tested with this new version and had their constructors modified to conform and are also passing.